code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -186,11 +191,13 @@
 	name = "\improper DromedaryCo packet"
 	desc = "A packet of six imported DromedaryCo cancer sticks. A label on the packaging reads, \"Wouldn't a slow death make a change?\""
 	icon_state = "Dpacket"
+	brand = "\improper Dromedary Co. cigarette"

```

Comment:
These descriptions aren't consistent with themselves, some have cigarette at the end and the others don't. I at least would prefer that they all use one or the other.